analyzers/src/SonarAnalyzer.Common/Rules/UnconditionalJumpStatementBase.cs

```diff
@@ -99,16 +110,16 @@ public List<TStatementSyntax> GetRuleViolations()
         {
             var ancestors = node
                 .Ancestors()
-                .TakeWhile(n => !this.rootExpression.Equals(n))
+                .TakeWhile(n => !rootExpression.Equals(n))
```

Comment:
Line 108/109: Empty line in the middle of method signature makes it hard to read